### DISCUSSION TITLE
How to set width of the menu popup in case of topNavigation()

### DISCUSSION THREAD

**Discussion by nishant91919** [filamentphp/filament#14186] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Please tell me how can I set the width of the popup in case of topNavigation(), as the text in my case is very long and it gets cut. Thanks in advance.

**Comment by nishant91919** [comment]
here is the screenshot.
![Screenshot 2024-09-09 211606](https://github.com/user-attachments/assets/37f91da5-3f62-4e30-8aaf-e8b7d50d0c7b)

**Reply by leandrocfe** [reply, marked as answer]
create a custom theme: https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

```css
.fi-topbar .fi-dropdown-panel {
    @apply min-w-96;
}
```

**Reply by nishant91919** [reply]
> create a custom theme: https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
> 
> ```css
> .fi-topbar .fi-dropdown-panel {
>     @apply min-w-96;
> }
> ```

Thank you so much...
